src/onboarding/welcome/Welcome.test.tsx

```diff
@@ -5,12 +5,15 @@ import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { firstOnboardingScreen } from 'src/onboarding/steps'
 import Welcome from 'src/onboarding/welcome/Welcome'
-import { Statsig } from 'statsig-react-native'
+import { updateStatsigUser } from 'src/statsig'
 import { createMockStore } from 'test/utils'
 import { mocked } from 'ts-jest/utils'
 
 jest.mock('src/onboarding/steps')
-jest.mock('statsig-react-native')
+jest.mock('src/statsig', () => ({
+  ...(jest.requireActual('src/statsig') as any),
```

Comment:
any reason to requireActual everything else here?

Reply:
yeah, we want `getExperimentParams` to return the default value rather than being a jest mock; tests fail otherwise